spec/mempool/cache.md

```diff
@@ -0,0 +1,36 @@
+# Mempool cache
+
+The *cache* stores the hashes of recently seen transactions. Its purpose is to avoid processing and
+adding duplicate transactions to the mempool.
+
+## Adding transactions to the cache
+
+A node that receives a transaction, before trying to add it to the mempool, puts it in the cache, if
```

Comment:
I tend to understand "receives a transaction" via network. Also via RPC, i.e., from clients?

Reply:
I would rephrase in a simpler way:

1. New transaction, check if it is on the cache
2. If it is, does nothing (what error message is produced?)
3. If it is not, add it to the cache to then start the process of adding it to the mempool.